tools/testsys/Test.toml.example

```diff
@@ -0,0 +1,103 @@
+# This is an example testing configuration for TestSys, the tool that
+# is used to validate Bottlerocket builds.
+
+# This section contains configuration details for all testing
+[test]
+# The repo from `Infra.toml` that should be used for testing. It may be
+# useful to create a repo in `Infra.toml` that contains the infrastructure
+# needed for testing
+repo = "default"
+# The registry containing alternate TestSys agent images
+testsys-images-registry = "public.ecr.aws/bottlerocket-test-system"
+
+# The URI for the EKS resource agent that should be used.
+# An individual agent's provided URI will be used even if `testsys-images-registry` is present.
+eks-resource-agent-image = "public.ecr.aws/bottlerocket-test-system/eks_resource_agent:v0.0.2"
+
+# Arguments passed at the command line will take precedence over environment variables and any
+# values specified in the configuration file.
+# Configurations will have the following resolution precedence:
+# '<VARIANT>' configuration in `Test.toml`
+# '<FAMILY>-<FLAVOR>' configuration in `Test.toml`
+# '<FAMILY>' configuration in `Test.toml`
+# '<PLATFORM>-<FLAVOR>' configuration in `Test.toml`
+# '<PLATFORM>' configuration in `Test.toml`
```

Comment:
I'm not sure if it should be demonstrated in this fashion, but do I understand this correctly?

(Also, is " configuration in `Test.toml`" actually required after each of these? If not, i.e. if it's obvious, then it would be less cluttered without that repeated suffix)

```suggestion
# '<VARIANT>'.ARCH
# '<VARIANT>'
# '<FAMILY>-<FLAVOR>'.ARCH
# '<FAMILY>-<FLAVOR>'
# '<FAMILY>'.ARCH
# '<FAMILY>'
# '<PLATFORM>-<FLAVOR>'.ARCH
# '<PLATFORM>-<FLAVOR>'
# '<PLATFORM>'.ARCH
# '<PLATFORM>'
```